biggraphite/drivers/elasticsearch.py

```diff
@@ -762,10 +769,15 @@ def __update_document(self, data, index, document_id):
             body=data
         )
 
-    def _create_search_query(self):
-        return elasticsearch_dsl.Search() \
+    def _create_search_query(self, start_time=None, end_time=None):
+        search = elasticsearch_dsl.Search() \
             .using(self.client) \
             .index("%s*" % self._index_prefix)
+        if start_time is not None:
+            search = search.filter('range', updated_on={"gte": start_time})
```

Comment:
you need to add the touch_ttl potential delay here

Reply:
Actual `updated_on` value should be greater or equal that the "real" one:

```
real_updated_on + touch_ttl >= updated_on >= real_updated_on
```
We want: 
```
real_updated_on >= start_time
```
so 
```
updated_on >= start_time
```
seems ok to me. 

We would get a bit more metrics than we should, at worse.

What would I miss here?

Reply:
Oh, understood. 

I added a comment to explain in the code. Will be checked in tests in a coming commit for this PR.